hoot-core/src/main/cpp/hoot/core/io/OgrReader.cpp

```diff
@@ -985,6 +1015,36 @@ Meters OgrReaderInternal::_parseCircularError(Tags& t)
   {
     bool ok;
     double a = t["accuracy"].toDouble(&ok);
+    if (!ok)
```

Comment:
The error:circular and corresponding accuracy tags should only ever store length values, not velocities.